### PR TITLE
5v5 Items

### PR DESCRIPTION
**Pull request by PierreAndreis**
Missing from last pr.

@seripap This should be asap

### CODE REVIEW

File: src/models/resources/items.js [hunk 1]
```diff
@@ -802,5 +802,53 @@ export default [
...
+  	"name": "Flaregun Loader"
+  },
+  {
+  	"serverName": "*Item_WarpGenerator*",
+  	"name": "Teleport Boots"
+  },
+  {
+  	"serverName": "Warp Generator",
+  	"name": "Teleport Boots"
+  },
```
Comment (seripap) [inline src/models/resources/items.js]: Can you remove the trailing comma? I'm not sure if I set up the correct Babel/stages to support trailing commas in array.

Comment (PierreAndreis) [reply src/models/resources/items.js]: @seripap done

**Comment by cklugewicz** [comment]
The 5v5 flare gun returns the same common name as the 3v3 flare gun, though the items are different. Can we add something to distinguish them, please?

**Comment by PierreAndreis** [comment]
They are currently different. 5v5 is Flaregun Loader

**Comment by cklugewicz** [comment]
Yep, saw that — removed my request already! XD

**Comment by PierreAndreis** [comment]
@seripap hey, 5v5 casual will be dropping likely tonight (PST). Is it possible to have this PR merged before?

**Comment by seripap** [comment]
Released on v1.3.8

**Comment by cklugewicz** [comment]
Hey, guys, `*Item_TotemEnhancer*` is actually `SuperScout 2000` in-game, not `SuperScout`

**Pull request merged**